editor/src/components/navigator/navigator-item/navigator-item.tsx

```diff
@@ -21,6 +21,9 @@ import { MetadataUtils } from '../../../core/model/element-metadata-utils'
 import { ThemeObject } from '../../../uuiui/styles/theme/theme-helpers'
 import { isFeatureEnabled } from '../../../utils/feature-switches'
 
+export const NavigatorItemTestId = (pathString: string): string =>
+  `NavigatorItemTestId-${pathString}`
```

Comment:
this could have been the trailing part of an `ElementPath` so that it mimics the `uid` and `testid` props with a prefix